specification/quantum/Quantum.Workspace/operations/items.tsp

```diff
@@ -0,0 +1,89 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+import "./operations.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+
+/** A key-value pair. */
+model KeyValue<Value = string> {
```

Comment:
would using the built in type of `Record<string>` not work?

Reply:
Sounds good. I'll do this in the next PR.